src/java/org/apache/cassandra/cql3/statements/CompositeCQLStatement.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.cql3.statements;
+
+import org.apache.cassandra.cql3.CQLStatement;
+
+public interface CompositeCQLStatement extends CQLStatement
```

Comment:
nit: What do you think about moving this and `ReturningCQLStatement` into `CQLStatement` itself, following the example of `SingleKeyspaceCqlStatement`? (I guess you could also name them `Composite` and `Returning` respectively, if you wanted to reference them like `CQLStatement.Composite`, but that definitely would make it inconsistent w/ `SingleKeyspaceCqlStatement`.)

Reply:
moved, but kept the `CQLStatement` postfix as it matches `SingleKeyspaceCqlStatement`.